Missing declaration file (image-lazy-load.service.d.ts)

After I installed ng2-image-lazy-load my typescript compilation failed:

```
[10:20:16] Starting 'compile:ts'...
[...]/node_modules/ng2-image-lazy-load/index.d.ts(2,15): error TS2307: Cannot find module './src/app/services/image-lazy-load.service'. undefined
[...]/node_modules/ng2-image-lazy-load/src/app/directives/image-lazy-load.directive.d.ts(4,62): error TS2307: Cannot find module '../services/image-lazy-load.service'. undefined
[10:20:20] TypeScript: 2 semantic errors
```

After a deeper look insight the 'src/app/services'-folder of ng2-image-lazy-load I noticed that there isn't a image-lazy-load.service.d.ts-file. 

I'm not sure if this is a mistake on my part or the missing declaration file is the reason. But after writing and adding the image-lazy-load.service.d.ts-file by myself locally the compilation succeeded.

npm --version: 3.7.3
node --version: 5.9.1

tsconfig.json

```
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "moduleResolution": "node",
    "sourceMap": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "removeComments": false,
    "noImplicitAny": true,
    "suppressImplicitAnyIndexErrors": true,
    "outDir": "public/dist"
  },
```

--- a/src/app/services/image-lazy-load.service.d.ts
+++ b/src/app/services/image-lazy-load.service.d.ts
@@ -1,0 +1,19 @@
+import {Http} from '@angular/http';
+import {WebWorkerStub} from './web-worker.service.stub';
+
+export declare class ImageLazyLoaderService {
+    http: Http;
+    worker: WebWorkerStub;
+    _config: Object;
+    imageCache: Object;
+    load(url: string, headers: Object): any;
+    loadViaWorker(url: string, headers: Object): any;
+    loadViaHttp(url: string, headers: Object): any;
+}
+
+export declare class IImageLazyLoadConfig {
+    headers?: any;
+    loadingClass?: string;
+    loadedClass?: string;
+    errorClass?: string;
+}

